modules/core/package.json

```diff
@@ -47,7 +47,7 @@
   },
   "dependencies": {
     "@bitgo/account-lib": "^2.15.0",
-    "@bitgo/statics": "^6.12.0",
+    "@bitgo/statics": "^6.13.0-rc.9",
```

Comment:
I've updated the version here because of the check-versions script running on the drone